packages/desktop-gui/src/styles/variables.scss

```diff
@@ -34,7 +34,7 @@ $screen-md: 1200px; // Medium screen / desktop
 $screen-lg: 1200px; // Large screen / wide desktop
 
 //** Padding applied to the modal body
-$modal-inner-padding:         15px !default;
+$modal-inner-padding:         0.9375rem !default;
```

Comment:
Bootstrap 4 changed the unit from px to rem. We need to use the same unit as bootstrap because bootstraps calculation will fail when mixing px and rem